org-code-javabuilder/media/src/main/java/org/code/media/AudioWriter.java

```diff
@@ -39,13 +39,22 @@ public void writeAudioSamples(double[] samples, double lengthSeconds) {
     this.writeAudioSamples(AudioUtils.truncateSamples(samples, lengthSeconds));
   }
 
-  public void writeAudioFile(String filename) throws SoundException, FileNotFoundException {
-    this.writeAudioSamples(SoundLoader.read(filename));
+  public void writeAudioAssetFile(String filename) throws SoundException, FileNotFoundException {
```

Comment:
should this be `writeAudioAssetFromFile`? If I'm understanding correctly it's writing to a stream, not a file, is that right?

Reply:
Good point, yep it's writing to a stream. How's `writeAudioFromAssetFile` and `writeAudioFromLocalFile` (to distinguish the two)?

Reply:
yep that's good!